metrics/metrics.go

```diff
@@ -108,3 +108,8 @@ func Start(metricsBind string, pprof http.Handler) {
 	s.Handler = mux
 	log.DefaultLogger.Warn("metrics", "listen finished", "err", s.ListenAndServe())
 }
+
+// GroupHandler provides metrics shared to other group members
+func GroupHandler() http.Handler {
```

Comment:
Can you write something in the doc explaining what this handler is exposing as info and on which path , and which port etc ?